src/ansiblelint/transformer.py

```diff
@@ -92,7 +96,12 @@ def run(self) -> None:
                 # We need a fresh YAML() instance for each load because ruamel.yaml
                 # stores intermediate state during load which could affect loading
                 # any other files. (Based on suggestion from ruamel.yaml author)
-                yaml = FormattedYAML()
+                yaml = FormattedYAML(
+                    # Ansible only uses YAML 1.1, but others files should use newer 1.2 (ruamel.yaml defaults to 1.2)
+                    version=(1, 1)
```

Comment:
There reason why I wanted to fallback to null was because that would make it make it future-proof for when ruamel.yaml will switch to versions such 1.3 or more.

What we want is to override the default for ansible file, and not touch it for others.

Reply:
Agreed. We should stick with `None` as the default value.